source/browseMode.py

```diff
@@ -1390,6 +1390,22 @@ def script_activateLongDesc(self,gesture):
 	# Translators: the description for the activateLongDescription script on browseMode documents.
 	script_activateLongDesc.__doc__=_("Shows the long description at this position if one is found.")
 
+	def script_activateAriaDetailsSummary(self, gesture):
+		info = self.makeTextInfo(textInfos.POSITION_CARET)
+		info.expand("character")
+		for field in reversed(info.getTextWithFields()):
+			if isinstance(field, textInfos.FieldCommand) and field.command == "controlStart":
+				states = field.field.get('states')
+				if states and controlTypes.STATE_HAS_ARIA_DETAILS in states:
+					speech.speakMessage(field.field['detailsSummary'])

```

Comment:
This should also output to braille. You can use the braille viewer to confirm. I think you can use `ui.message` for that.